keps/sig-auth/20191101-add-request-id-to-k8s-logs.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Add Request-ID to each Kubernetes component log
+authors:
+ - "@hase1128"
+ - "@sshukun"
+ - "@furukawa3"
+ - "@vanou"
+owning-sig: sig-instrumentation
+participating-sigs:
+reviewers:
+ - TBD
+approvers:
+ - TBD
+editor: TBD
+creation-date: 2019-11-01
+last-updated: 2020-03-17
+status: provisional
+---
+
+# Add Request-ID to each Kubernetes component log
+
+## Table of Contents
+
+<!-- toc -->
+ - [Summary](#summary)
+ - [Motivation](#motivation)
+   - [Target User](#target-user)
+   - [Target User's objective](#target-users-objective)
+   - [Case 1](#case-2)
+   - [Case 2](#case-1)
+   - [Goals](#goals)
+   - [Non-Goals](#non-goals)
+ - [Proposal](#proposal)
+   - [Basic policy of Request-ID](#basic-policy-of-request-id)
+   - [Design Overview](#design-overview)
+     - [Design of Propagate Request-ID](#design-of-propagate-request-id)
+     - [Design of Export Request-ID](#design-of-export-request-id)
+     - [Design overview of Control Request-ID](#design-overview-of-control-request-id)
+     - [Detail design of Control Request-ID](#detail-design-of-control-request-id)
+ - [Test Plan](#test-plan)
+ - [Migration / Graduation Criteria](#migration--graduation-criteria)
+   - [Alpha](#alpha)
+   - [Beta](#beta)
+   - [GA](#ga)
+
+<!-- /toc -->
+
+## Summary
+
+This KEP proposes a new unique logging meta-data into all Kubernetes logs. It makes us
```

Comment:
It looks like you have a well-defined list of places you would like to add this log below.  Maybe say here that you are adding metadata to "key" kuberntes logs, rather than "all" logs.  "all" logs would be an enormous undertaking and may not be realistic.

Reply:
yes, I modified the word "all".